src/index.ts

```diff
@@ -6,17 +6,17 @@ import { CollectionConfig, GlobalConfig } from "payload/types";
 type CollectionOrGlobalConfig = CollectionConfig | GlobalConfig;
 
 interface PluginCollectionOrGlobalConfig {
-    previewUrl?: string;
+    previewUrl?: (locale: string) => string;
```

Comment:
Just an idea:

if we pass an object instead of a string to the previewUrl function, we could add more params at some point without breaking the code.

Something like this:

```
type PreviewUrlFn = (params: { locale: string; }) => string;

previewUrl?: PreviewUrlFn;
```
```
{
  ...
  previewUrl: params => `http://localhost:8080/${params.locale}`,
}